Assets/MixedRealityToolkit-SDK/Features/Input/GazeProvider.cs

```diff
@@ -150,6 +150,15 @@ public InternalGazePointer(GazeProvider gazeProvider, string pointerName, IMixed
             /// <inheritdoc />
             public override IMixedRealityInputSource InputSourceParent { get; protected set; }
 
+            private float pointerExtent;
+
+            /// <inheritdoc />
+            public override float PointerExtent
```

Comment:
What's the benefit of overriding here, when we don't do anything differently than the base class?

Reply:
because we're setting it in the constructor, and because it's virtual it'll cause some unintended side effects.

Reply:
Isn't that just for virtual methods called in base constructors?

Reply:
Any virtual member call in any constructor.

Essentually if you set a virtual member in a constructor, it's calling on a derived member that might not be initialized yet.

Reply:
https://stackoverflow.com/questions/119506/virtual-member-call-in-a-constructor